iap_watcher/iap_watcher.py

```diff
@@ -55,6 +55,12 @@ def Main(argv, watcher=None, loop_watcher=True):
   logger = logging.getLogger()
   logger.setLevel(logging.INFO)
 
+  # This ensures we have fresh keys at container start. Doing it here because
+  # Docker doesn't support multiple CMD/ENTRYPOINT statements in Dockerfiles.
```

Comment:
You might be able to use https://docs.docker.com/engine/userguide/eng-image/multistage-build/
although admittedly I haven't dug much into the Dockerfile format.

(you might also be blocked by the version of Docker used in our jenkins instance)

Reply:
I'd prefer not to invest too heavily here because I think the long-term future of key fetching is to move into the nginx module (that's nontrivial work however).

Reply:
SGTM